MAINTAINERS.yml

```diff
@@ -0,0 +1,1009 @@
+#         Plain-English description. Describe what the system is about, from an
+#         outsider's perspective.
+#
+#
+# All areas must have a 'status' key and a 'files' and/or 'files-regex' key.
+# The other keys are optional.
+#
+#
+# Workflow
+# ########
+#
+# Ideally, any file in the tree will be covered by some area.
+#
+# When a GitHub pull request is sent, this happens:
+#
+#     * Users mentioned in 'maintainers' and 'collaborators' are added as
+#       reviewers to the pull request
+#
+#     * The labels listed in 'labels' are automatically added to the pull
+#       request
+#
+#     * The bot posts this comment:
+#
+#         This PR affects the following areas:
+#         <area name>:
+#           Status: ...
+#           Maintainers: <list of maintainers>
+#           Collaborators: <list of sub-maintainers>
+#           CC: <list of informed GitHub users, like @johndoe>
+#
+#         <area name>:
+#           ...
+#
+#     * An e-mail is sent to the addresses in 'inform'
+#
+# Changes to MAINTAINERS.yml need to be approved as follows:
+#
+#     * Changing the 'maintainers' for an area needs approval from the
+#       technical steering committee
+#
+#     * Changing the 'collaborators' lines requires the maintainer and
+#       collaborators of that area to agree (or vote on it)
+
+# Areas are sorted by name
+
+"ARC arch":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - ruuddw
+    collaborators:
+        - vonhust
+    files:
+        - arch/arc/
+        - include/arch/arc/
+    labels:
+        - "area: ARC"
+
+"ARM arch":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - ioannisg
+    collaborators:
+        - MaureenHelm
+        - galak
+        - ioannisg
+    files:
+        - arch/arm/
+        - include/arch/arm/
+    labels:
+        - "area: ARM"
+
+Bluetooth:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - jhedberg
+    collaborators:
+        - joerchan
+        - Vudentz
+    files:
+        - doc/reference/bluetooth/
+        - doc/guides/bluetooth/
+        - drivers/bluetooth/
+        - include/bluetooth/
+        - include/drivers/bluetooth/
+        - samples/bluetooth/
+        - subsys/bluetooth/
+        - tests/bluetooth/
+    labels:
+        - "area: Bluetooth"
+
+"Bluetooth controller":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - cvinayak
+    collaborators:
+        - carlescufi
+        - thoh-ot
+    files:
+        - subsys/bluetooth/controller/
+
+"Bluetooth Mesh":
+    # Provisional entry, to be corrected
+    status: maintained
+    collaborators:
+        - trond-snekvik
+    files:
+        - subsys/bluetooth/mesh/
+    labels:
+        - "area: Bluetooth Mesh"
+
+"Build system":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - SebastianBoe
```

Comment:
nashif is a co-maintainer and should be listed if I understand this correctly.

Reply:
Otherwise looks good.

Reply:
"collaborators", roger will do